src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -584,6 +585,7 @@ public void StopStake()
             }
 
             this.logger.LogTrace("Wallet total staking balance is {0}.", new Money(utxoStakeDescriptions.Sum(d => d.TxOut.Value)));
+            this.logger.LogTrace("(-)");
```

Comment:
` this.logger.LogTrace("(-):*.{0}={1}", nameof(utxoStakeDescriptions.Count), utxoStakeDescriptions.Count);`